homeassistant/components/google_cloud/tts.py

```diff
@@ -22,7 +22,10 @@
 CONF_PROFILES = "profiles"
 
 SUPPORTED_LANGUAGES = [
+    "ar-XA",
+    "bn-IN",
     "cs-CZ",
+    "cmn-CN",
```

Comment:
If you want this list to be alphabetic, this should be switched with `cs-CZ`

Reply:
Yep, thx! Pushed.